CHANGES

```diff
@@ -42,6 +42,10 @@
   *) Support for SSL_OP_NO_ENCRYPT_THEN_MAC in SSL_CONF_cmd.
      [Emilia Käsper]
 
+  *) The RSA "null" method, which was partially supported to avoid patent
+     issues, has been replaced to always returns NULL.
+     [Rich Salz]
```

Comment:
I have my doubts about how wise it is to backport this part of the change to 1.1.0. There doesn't seem any pressing need to do so (it doesn't fix any bugs).